setup.py

```diff
@@ -24,7 +24,7 @@
     url="https://github.com/pulp/pulp-cli",
     version="0.10.0.dev",
     packages=packages,
-    package_data={package: ["py.typed"] for package in plugin_packages},
+    package_data={package: ["py.typed"] for package in plugins},
```

Comment:
This is missing the "common" package due to line 8.
Also shouldn't it be `packages`, maybe without the `"pytest_pulp_cli"`.

Reply:
mein name ist hase!

upstream uses the same:
https://github.com/pulp/pulp-cli/blob/9165a5e5b9e22f3ec13d575dd2ab2dc05246a9d2/setup.py#L24

```python
plugin_packages = find_namespace_packages(include=["pulpcore.cli.*"])
packages = plugin_packages + ["pytest_pulp_cli"]
plugin_packages.remove("pulpcore.cli.common")
…
    package_data={package: ["py.typed"] for package in plugin_packages},
```

Reply:
Not exactly. `plugins` (branch) contains only the last name, while `plugin_packages` (upstream) has the full names.

They both miss the `common` not-so-much-a-plugin package...